### PR TITLE
Filled requirements to project

### PR DESCRIPTION
**Pull request by andrianarivo**
- Document in a professional way
- new branch according to the GitHub flow rules.
- Add a .gitignore file.
- Add node_modules to gitignore file.
- Set up a linter for HTML and CSS
- Added descriptive README file to project
- Added 3 files (each one in a separate commit):
  - A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
  - A .css file that includes one class with styles for your header (kind of cyan color).
  - A test.md file with the text "This file should be ignored by git". The file is not present in git repo.
- Opened a pull request.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```
Comment (joel-kalema) [inline .github/workflows/linters.yml:2]: - [ ] As I can see, you did a good job @andrianarivo 😃 , but make sure that there are [no linter errors](https://github.com/microverseinc/linters-config) as it is [required](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=Make%20sure%20that%20there%20are%20no%20linter%20errors.). you may check is [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to fix it and keep doing great work 👍

linters

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 

- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] You did great work on your commit message 😃, but to make it more descriptive, you should follow some structures, inside of saying `update readme and gitignore to requirements` you should say `Modified the README file` and make sure you capitalize the first letter 👍 , you can check this [link](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/) for more information.

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (andrianarivo) [reply .github/workflows/linters.yml:2]: @joel-kalema thank you for reviewing this issue. Yes, in fact it was a lighthouse issue that I solved using this discourse thread: [Lighthouse Report failing on Awesome books with ES6 activity - Linters & Stickler - Microverse](https://questions.microverse.org/t/lighthouse-report-failing-on-awesome-books-with-es6-activity/4773)

Have a good day and happy coding

**Review by divinecharlotte — APPROVED** [review]
# Hello @andrianarivo     😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 
### To Highlight 💯

- No linters errors ✔️
- Descriptive PR ✔️
- Descriptive commit messages 👍 





## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by andrianarivo** [comment]
@divinecharlotte Thanks ! ^^

**Pull request merged**